src/main/proto/wfa/virtual_people/common/model.proto

```diff
@@ -203,6 +204,20 @@ message ConditionalMerge {
   optional bool pass_through_non_matches = 2;
 }
 
+// If the condition is satisfied, applies all the assignments, each of which is
+// to copy the value from source_field to target_field.
+// Does nothing if the condition is not satisfied.
+message ConditionalAssignment {
```

Comment:
Are there any examples of this anywhere?

Reply:
Added in comments.